ui/src/app/views/dataset-page/component.tsx

```diff
@@ -83,7 +83,8 @@ export const ValueListItem: React.FunctionComponent <
   {domainValue: DomainValue, onSelect: Function, checked: boolean}> =
   ({domainValue, onSelect, checked}) => {
     return <div style={{display: 'flex', color: 'black', height: '1.2rem'}}>
-      <input type='checkbox' value={domainValue.value} onChange={() => onSelect()}
+      <input type='checkbox' value={domainValue.value} onClick={() => onSelect()}
```

Comment:
You shouldn't need to add a click handler for this. Just, above, in the tests, pass 'change' in as the arg to simulate?

Reply:
I was actually hoping to find solution for input type checkbox to remove onChange, technically change would work but since its a click event i was trying to let the test be. I am going to try some other thing if that will not work i will revert this and change test to have "change" rather than click

Reply:
Ok so far what i have read since onChange and onClick behaves different only for IE i'll revert this and use 'change' in test